pkg/seeds/seeds.go

```diff
@@ -221,10 +221,14 @@ func createArch(existingArch *string) string {
 
 const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 
-func RandStringBytes(n int) string {
+func RandStringWithTable(n int, lookup string) string {
```

Comment:
what does "WithTable" mean here? It looks like this returns a string using letters randomly grabbed from lookup. Is "RandStringWithChars" a better fit?

Reply:
I mean `RandStringWithLookupTable`, but I like more `RandStringWithChars` that is shorter and explanatory. Thanks!